Packs/CortexResponseAndRemediation/Playbooks/playbook-Uncommon_execution_of_ODBCConf.yml

```diff
@@ -0,0 +1,715 @@
+    task:
+      id: ccc98587-c43d-4666-8b85-c27092f73e1a
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+      - "11"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: d6d828b0-4213-478a-84e4-56ab20a4ce74
+    type: regular
+    task:
+      id: d6d828b0-4213-478a-84e4-56ab20a4ce74
+      version: -1
+      name: Check if the causality process is prevalent
+      description: Get the prevalence of a process, identified by process_name.
+      script: '|||core-get-process-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      process_name:
+        simple: ${alert.cgoname}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": -100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: a30d9322-0cf5-469f-88cf-c6c63d905b08
+    type: condition
+    task:
+      id: a30d9322-0cf5-469f-88cf-c6c63d905b08
+      version: -1
+      name: Check if the causality process is prevalent and not signed
+      description: This task checks if the causality process is prevalent and not signed
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "12"
+      Malicious:
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isEmpty
+          left:
+            value:
+              simple: alert.cgosigner
+            iscontext: true
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Core.AnalyticsPrevalence.Process.value
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 765fe8d2-bdd1-4be4-8a98-48c82c984a70
+    type: title
+    task:
+      id: 765fe8d2-bdd1-4be4-8a98-48c82c984a70
+      version: -1
+      name: Remediation
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4.
```

Comment:
Terminate a process tree by its causality ID. Available for Cortex XSIAM 2.4 and above.